src/main/java/Duke.java

```diff
@@ -1,10 +1,250 @@
+        System.out.println("\tHello! I'm Duke");
+        System.out.println("\tWhat can I do for you?");
+        System.out.println(DOTTED_LINE);
+    }
+
+    private static void loadData() {
+        try {
+            File directory = new File(DIR);
+            directory.mkdir();
+
+            File file = new File(FILEPATH);
+            // if file already exists, read it
+            if (!file.createNewFile()) {
+                Scanner reader = new Scanner(file);
+                while (reader.hasNext()) {
+                    handleInput(reader.nextLine(), false);
+                }
+            }
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+
+    private static void readInputs() {
+        Scanner scanner = new Scanner(System.in);
+        String input = scanner.nextLine();
+        while (!input.equalsIgnoreCase(EXIT_COMMAND)) {
+            handleInput(input, true);
+            input = scanner.nextLine();
+        }
+    }
+
+    private static void addInputToFile(String input) throws IOException {
+        // create a FileWriter in append mode
+        FileWriter fw = new FileWriter(FILEPATH, true);
+        fw.write(input + System.lineSeparator());
+        fw.close();
+    }
+
+    private static void handleInput(String input, boolean finishSetup) {
+        if (finishSetup) {
+            System.out.println(DOTTED_LINE);
+        }
+        try {
+            if (input.trim().equalsIgnoreCase(LIST_COMMAND)) {
+                executeCommand(input, LIST_COMMAND, finishSetup);
+            } else if (input.startsWith(DONE_COMMAND)) {
+                executeCommand(input, DONE_COMMAND, finishSetup);
+            } else if (input.startsWith(DELETE_COMMAND)) {
+                executeCommand(input, DELETE_COMMAND, finishSetup);
+            } else if (input.startsWith(TODO_COMMAND)) {
+                executeCommand(input, TODO_COMMAND, finishSetup);
+            } else if (input.startsWith(DEADLINE_COMMAND)) {
+                executeCommand(input, DEADLINE_COMMAND, finishSetup);
+            } else if (input.startsWith(EVENT_COMMAND)) {
+                executeCommand(input, EVENT_COMMAND, finishSetup);
+            } else {
+                System.out.println("\t☹ OOPS!!! I'm sorry, but I don't know what that means :(");
+            }
+        } catch (NumberFormatException e) {
+            System.out.println("\tPlease enter the task number too!");
+        } catch (InvalidIndexException e) {
+            System.out.println("\t☹ OOPS!!! I can't find this task");
+        } catch (EmptyNameException e) {
+            System.out.println("\t☹ OOPS!!! The description of a todo cannot be empty.");
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+        if (finishSetup) {
+            System.out.println(DOTTED_LINE);
+        }
+    }
+
+    private static void executeCommand(String input, String command, boolean finishSetup)
```

Comment:
It might help to improve readability if the boolean variable was renamed to sound more like a boolean than a statement